uaclient/tests/test_status.py

```diff
@@ -795,6 +806,7 @@ def test_expires_handled_appropriately(
         _m_remove_notice,
         all_resources_available,
         FakeConfig,
+        tmpdir,
```

Comment:
Why do we need this fixture here ?

Reply:
I was testing something, forgot to cleanup.